src/main/scala/is/hail/stats/RegressionUtils.scala

```diff
@@ -58,49 +59,45 @@ object RegressionUtils {
   }
 
   // IndexedSeq indexed by samples, Array by annotations
-  def getSampleAnnotations(vds: MatrixTable, annots: Array[String], ec: EvalContext): IndexedSeq[Array[Option[Double]]] = {
-    val aQs = annots.map(parseFloat64Expr(_, ec))
-
-    vds.colValues.value.map { sa =>
-      ec.set(0, sa)
-      aQs.map { aQ =>
-        val a = aQ()
-        if (a != null)
-          Some(a: Double)
-        else
-          None
-      }
+  def getSampleAnnotations(mt: MatrixTable, names: Array[String]): IndexedSeq[Array[Option[Double]]] = {
```

Comment:
I hate annotations. Mind changing to `getColumnDoubles`?

And the remark should say `IndexedSeq indexed by samples, Array by field`